IPython/core/interactiveshell.py

```diff
@@ -354,9 +354,8 @@ def _exiter_default(self):
     @observe('prompt_in1', 'prompt_in2', 'prompt_out', 'prompt_pad_left')
     def _prompt_trait_changed(self, change):
         name = change['name']
-        warn("InteractiveShell.{name} is deprecated since IPython 4.0 and ignored since 5.0, set TerminalInteractiveShell.prompts object directly.".format(
-                name=name)
-        )
+        warn("InteractiveShell.{name} is deprecated since IPython 4.0 and ignored since 5.0, set TerminalInteractiveShell.prompts object directly.".format(name=name), stacklevel=2)
```

Comment:
@srinivasreddy could you back out this change for now? I'd rather it was wrong in the obvious simple way than wrong in a new and confusing way. Thanks!